packages/clima_ui/lib/widgets/settings/link_text_span.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/gestures.dart';
```

Comment:
Don't put this under `settings`; put it directly under `widgets`. There's nothing specific to the settings page here.

Reply:
Do it yourself.

Reply:
Oh, you actually changed it yourself!